localnet_test.go

```diff
@@ -49,7 +49,7 @@ var _ = Describe("Local Testnet", func() {
 			fmt.Printf("./build/emd  q market instrument eeur echf --node %v\n\n", node)
 
 			fmt.Println(" -- Run this command for a pre-configured local environment:")
-			fmt.Printf("EM_NODE=%v EM_HOME=%v EM_KEYRING_BACKEND=test EM_CHAIN_ID=%v PS1=\"$PS1 [testnet] \" sh\n", node, keystore, chainid)
+			fmt.Printf("EMD_NODE=%v EMD_HOME=%v EMD_KEYRING_BACKEND=test EMD_CHAIN_ID=%v EMD_KEYRING_DIR=%v PS1=\"$PS1 [testnet] $\" sh\n", node, keystore, chainid, keystore)
```

Comment:
Could we exclude the $PS1 variable? It's incompatible in my case and puts out incoherent results in my case.
So it would be like this. Or we could add prompt escape characters i.e., ```\w```
```Go
fmt.Printf("EMD_NODE=%v EMD_HOME=%v EMD_KEYRING_BACKEND=test EMD_CHAIN_ID=%v EMD_KEYRING_DIR=%v PS1=\"[testnet] $\" sh\n", node, keystore, chainid, keystore)
```

Reply:
I found a better overall workaround. Can I suggest bash instead of sh? You still get the **changed** shell feel, colors, and works for me too. Ok?

Reply:
Sounds good.